templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -22,25 +22,6 @@ objects:
       app: ansible-service-broker
       service: asb
 
-- apiVersion: v1
```

Comment:
Just noting, these changes will need to be reflected in openshift-ansible once this is officially supported